task_alert/README.rst

```diff
@@ -0,0 +1,28 @@
+Plan task as a reminder for any date field
+==========================================
+Plan when a task should be created based on a nearing deadline. Any date field can be used. How much time in
+advance the task should be created can be specified.
+
+Description
+-----------
+Create a project to which the tasks will be related:
+
+	**Project > Project > projects**
+
+Task alerts should be defined in:
+
+	**Project > Configuration Task Alerts**
+
+Add a name and a description for the task, who will receive the task.
+
+A cron job named "Create alert tasks" will be created by installing this module. This job can be found via:
+
+	**Settings > Technical > Automation > Scheduled Actions**
+
+This job will run daily by default.
+
+|
+
+**Contact us:**
+
+When you have any remark about this module, please let us know on http://www.onestein.eu/feedback
```

Comment:
But that's why there is an "Contributors" section. There you have an email.
I agree i will be a nice discussion.
@dreispt @elicoidal Would be correct for example to add in "Cotnributors" section add a line like:

- Onestein Feedback (info@onestein.nl)

??